src/Equinor.ProCoSys.IPO.Domain/AggregateModels/ProjectAggregate/Project.cs

```diff
@@ -31,6 +32,7 @@ public Project(string plant, string name, string description)
         public int CreatedById { get; private set; }
         public DateTime? ModifiedAtUtc { get; private set; }
         public int? ModifiedById { get; private set; }
+        public Guid Guid { get; set; } = Guid.Empty;

```

Comment:
remove init Guid.Empty;
Add a Todo // make Guid private after FillGuids is completed